Bottom: d1b07f8f08aa243aa8b984a5871f306fe83621f4
Top:    919e8b28ff69f069f4d92811dc167b986d0d73df
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 13:53:03 +0000

Refresh of wip-on-quad-edge-play-STEP

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 449d44a..24082a4 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -1,12 +1,11 @@
 #include <stdio.h>
 //#include <stdlib.h>
-#include <stdbool.h>
+//#include <stdbool.h>
 
 #include <glib.h>
 
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
-#include "hid/common/contour3d.h"
 #include "hid/common/appearance.h"
 #include "hid/common/face3d.h"
 //#include "hid/common/edge3d.h"
